Patch to include Random123 from Easybuild, rather then the external source
Author: J. Sassmannshausen (Imperial College London/UK)
diff --git a/fresnel-0.13.7.orig/CMakeLists.txt b/fresnel-0.13.7/CMakeLists.txt
index 8151fe6..97bb6a5 100644
--- a/fresnel-0.13.7.orig/CMakeLists.txt
+++ b/fresnel-0.13.7/CMakeLists.txt
@@ -132,8 +132,8 @@ endif()
 
 #############################################################################################
 ### Add project subdirs
-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/extern/random123/include)
-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/extern/random123/examples)
+include_directories($ENV{EBROOTRANDOM123}/include)
+include_directories($ENV{EBROOTRANDOM123}/examples)
 
 add_subdirectory(fresnel)
 
